gax-grpc/src/test/java/com/google/api/gax/grpc/testing/LocalChannelProvider.java

```diff
@@ -105,6 +127,53 @@ public String getTransportName() {
 
   /** Creates a LocalChannelProvider. */
   public static LocalChannelProvider create(String addressString) {
-    return new LocalChannelProvider(addressString);
+    return new LocalChannelProvider(new LocalAddress(addressString), null);
+  }
+
+  public boolean validateSentHeader(String headerKey, Pattern headerPattern) {
```

Comment:
I would expect a "validateX" function to throw an exception if the validation failed; maybe instead call this "isSentHeaderValid"

Reply:
renamed to `isHeaderSent()`